ViewLayer/ViewContainer.h

```diff
@@ -2,17 +2,21 @@
 
 #include <QScopedPointer>
 
-class I_DisplayDashboardUI;
+enum Mode { RACE, DISPLAY };
```

Comment:
Actually I think the best answer is this one http://stackoverflow.com/a/12865221